lib/utils/Logger.js

```diff
@@ -163,17 +163,12 @@ Logger.prototype.data = function(data) {
  */
 Logger.prototype.result = function (result) {
 
-    // if result is undefined, null or empty don't print it
-    if(typeof result === 'undefined' || (typeof result === 'object' && !result) || (typeof result === 'string' && result.length === 0)) {
-        return;
```

Comment:
Instead of just printing nothing, can we get something more meaningful like `(empty)` or `""`